src/droplet.F

```diff
@@ -502,7 +502,7 @@ subroutine droplet_driver(dt,dbldt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,
     do k=kb,ke
        do j=jb,je
           do i=ib,ie
-             dpten(i,j,k,5) = dpten(i,j,k,5)*pi0(i,j,k)
+             dpten(i,j,k,1) = dpten(i,j,k,1)*pi0(i,j,k)
```

Comment:
Is this a bug fix or just due to a reorder of `dpten` index?

Reply:
Bug fix! George pointed it out last week

Reply:
Thanks for the clarification.